content/departments/sales/tools/referral_and_reseller_policy.md

```diff
@@ -0,0 +1,85 @@
+**<span style="text-decoration:underline;">Referral Program Summary</span>**
+
+- <span style="text-decoration:underline;">Program Purpose</span>: The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will enthusiastically pay them for accelerating our customer acquisition.
```

Comment:
Underlines should be avoided in handbook pages, since these go on the web, and on the web underlines usually represent links. Instead, use only bold or italic.

After conversion, this will be only wrapped in asteriks, instead of the verbose `<span style="text-decoration:underline;">`.